src/styles/Blocks/calendar.scss

```diff
@@ -0,0 +1,59 @@
+$day-size: 100px;
+$calendar_gap: 1px;
+$calendar_padding: 10px;
+$days:
+  "mon" 0,
+  "tue" 1,
+  "wed" 2,
+  "thu" 3,
+  "fri" 4,
+  "sat" 5,
+  "sun" 6;
```

Comment:
you declared these variables in a separate file

Reply:
About variables. A added them to other files because they did`t work in only variables file. I wrote it in the titel.